### PR TITLE
add parts of styling to main, login and signup

### PR DESCRIPTION
**Pull request by MRzsztk**

### CODE REVIEW

File: src/pages/Login/style.css [hunk 1]
```diff
@@ -1,6 +1,39 @@
-.login-form{
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+#root {
+    font-family: Roboto;
+}
+.container {
```
Comment (CarlosZiegler) [inline src/pages/Login/style.css:5]: If you have a global style, add in src/index.css

File: src/pages/Signup/style.css [hunk 2]
```diff
@@ -1,6 +1,44 @@
...
+.signup-form input {
+    height: 7vh;
+    margin: 7px;
+    border-radius: 15px;
+    border: 1px solid gray;
+}
+
+.signup-form button {
+    box-sizing: border-box;
+    height: 7vh;
```
Comment (CarlosZiegler) [inline src/pages/Signup/style.css:36]: not use vh for button, you can use px for this element.

Comment (MRzsztk) [reply src/pages/Signup/style.css:36]: OK understood!

File: src/pages/Login/style.css (hunk 1 above)
Comment (MRzsztk) [reply src/pages/Login/style.css:5]: OK!

**Pull request closed without merging**